presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/TestFileResourceGroupConfigurationManager.java

```diff
@@ -59,10 +85,19 @@ public void testConfiguration()
 
     private FileResourceGroupConfigurationManager parse(String fileName)
     {
-        String path = this.getClass().getClassLoader().getResource(fileName).getPath();
         ResourceGroupConfig config = new ResourceGroupConfig();
-        config.setConfigFile(path);
-        return new FileResourceGroupConfigurationManager(config, new ObjectMapperProvider().get());
+        config.setConfigFile(getResourceFilePath(fileName));
+        return new FileResourceGroupConfigurationManager(
+                (poolId, listener) -> {
+                    // no-op
```

Comment:
you don't need this comment... this patter is very common. Instead put both brackets on one line